src/models/entities/entity.ts

```diff
@@ -0,0 +1,7 @@
+export interface Entity {
+  id: number;
```

Comment:
@madhav-devslane Same reference from [this comment](https://github.com/devslane/nextjs-redux-tailwind-boilerplate/pull/2#discussion_r830929920)